src/game/AI/ScriptDevAI/scripts/kalimdor/dustwallow_marsh.cpp

```diff
@@ -1000,6 +1000,126 @@ UnitAI* GetAI_mob_invis_firework_helper(Creature* pCreature)
     return new mob_invis_firework_helper(pCreature);
 }
 
+/*######
+## npc_theramore_practicing_guard
+######*/
+enum
+{
+    NPC_THERAMORE_COMBAT_DUMMY = 4952,
+};
+
+struct npc_theramore_practicing_guardAI : public ScriptedAI
+{
+    npc_theramore_practicing_guardAI(Creature* pCreature) : ScriptedAI(pCreature)
+    {
+        Reset();
+        m_creature->SetNoCallAssistance(true);
+        m_creature->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_IMMUNE_TO_PLAYER);
+    }
+
+    uint32 m_attackTimer;
+    uint32 m_breakTimer;
+    uint32 m_sitTimer;
+    bool m_bisAttacking;
+    bool m_bsitDown, m_bstandUp;
+    Creature* attackableDummy;
+
+    void Reset() override
+    {
+        SetRootSelf(true);
+        m_attackTimer = 130 * IN_MILLISECONDS;
+        m_breakTimer = 20 * IN_MILLISECONDS;
+        m_bisAttacking = false;
+        m_bsitDown = false;
+        m_bstandUp = false;
+        attackableDummy = GetClosestCreatureWithEntry(m_creature, NPC_THERAMORE_COMBAT_DUMMY, 2.f);
+    }
+
+    void UpdateAI(const uint32 diff) override
+    {
+        if (attackableDummy && m_creature->GetVictim() && m_creature->GetVictim() != attackableDummy)
```

Comment:
This exists because
```cpp
m_creature->SetNoCallAssistance(true);
```
apparently has no effect, meaning that when one Theramore Practicing Guard starts attacking its dummy, the other one also reacts and wants to attack that dummy. This line is preventing this from happening, but you can still hear the shout NPCs do when they enter combat.

If there is a more elegant solution to this, I'm very open to it